compiler/aarch64/codegen/OMRInstOpCode.enum

```diff
@@ -910,6 +910,12 @@
 		vrev64_16b,                                              	/* 0x4E200800	REV64    	 */
 		vrev64_8h,                                               	/* 0x4E600800	REV64    	 */
 		vrev64_4s,                                               	/* 0x4EA00800	REV64    	 */
+		vxtn_8b,                                                 	/* 0x0E212800	XTN     	 */
+		vxtn_4h,                                                 	/* 0x0E612800	XTN     	 */
+		vxtn_2s,                                                 	/* 0x0EA12800	XTN     	 */
+		vxtn2_16b,                                               	/* 0x4E212800	XTN     	 */
+		vxtn2_8h,                                                	/* 0x4E612800	XTN     	 */
+		vxtn2_4s,                                                	/* 0x4EA12800	XTN     	 */
```

Comment:
The comment in these three lines has to be `XTN2`.

Reply:
Fixed it.